python/requirements.txt

```diff
@@ -48,7 +48,7 @@ scikit-image==0.21.0
 prometheus_client>=0.7.1
 pandas
 tensorboardX
-aiohttp>=3.7,!=3.9.0
+aiohttp>=3.7,<3.9
```

Comment:
w00t, we need to make it possible to go above 3.9 so that ray can be installed with python 3.12; this change has been here when the tests pass previously too i think

Reply:
based on the slack thread, perhaps change this to `<3.10.0`, and then you don't need other changes to the requirements file

Reply:
Let me make the change and test it.